### PR TITLE
Export library versions

### PR DESCRIPTION
**Pull request by tnorling**
Exports version from each library

Addresses #4122

### CODE REVIEW

**Review by jasonnutter — APPROVED** [review]

**Review by jo-arroyo — APPROVED** [review]

File: lib/msal-common/src/index.ts [hunk 1]
```diff
@@ -93,3 +93,4 @@ export { TimeUtils } from "./utils/TimeUtils";
 export { ServerTelemetryManager } from "./telemetry/server/ServerTelemetryManager";
 export { ServerTelemetryRequest } from "./telemetry/server/ServerTelemetryRequest";
 export { PopTokenGenerator } from "./crypto/PopTokenGenerator";
+export { version } from "./packageMetadata";
```
Comment (sameerag) [inline lib/msal-common/src/index.ts:96]: Do we need this?

Comment (tnorling) [reply lib/msal-common/src/index.ts:96]: We might want it later. At worst it's a consistency item, this is a good thing to have available.

Comment (sameerag) [reply lib/msal-common/src/index.ts:96]: Alright. Approving.

**Review by hectormmg — APPROVED** [review]
Lgtm

**Pull request merged**